src/components/upload.vue

```diff
@@ -113,6 +113,15 @@ export default {
       const formData = new FormData();
       const { name, size, type } = file;
 
+      if (size !== -1 && size > this.$store.state.serverInfo.maxUploadSize) {
+        this.$events.emit("warning", {
+          notify: this.$t("upload_exceeds_max_size", { filename: name }),
+          color: "orange"
```

Comment:
```suggestion
          color: "warning"
```

Can we change this to use the warning color? That will make sure it's consistent if the user changes the warning color to something !== orange 🙂

Reply:
Sure, I didn't know it was possible to customize the warning color.